frontend/src/v4/routes/components/criteriaField/criteriaField.styles.ts

```diff
@@ -159,8 +159,7 @@ const emptySelectFieldValueStyleV4 = css`
 	opacity: 0.42;
 `;
 const emptySelectFieldValueStyleV5 = css`
-	/* TODO - fix after new palette is released */
-	color: #C1C8D5;
+	color: ${({ theme }) => theme.palette.base.lighter};
```

Comment:
Does this thing works in v4? theme.palette.base.lighter is from v5, right

Reply:
I have tested and they both work